js/capture.js

```diff
@@ -1,5 +1,5 @@
-/* eslint no-console: "off" */
-/* eslint no-unused-vars: "off" */
+/* eslint no-unused-vars: "off"*/
```

Comment:
My hunch is capture needs to be added as a global in the eslint file. We probably want a more explicit name eventually too that won't collide with other library globals.